
def func(n):
    try:
        print('hello')
    except Exception:
        pass
